saltproc/app.py

```diff
@@ -240,9 +240,9 @@ def _create_depcode_object(depcode_input):
     """Helper function for `run()` """
     codename = depcode_input['codename']
     if codename == 'serpent':
-        depcode = DepcodeSerpent
+        depcode = SerpentDepcode
```

Comment:
I guess this is fine, lol. Why rearrange?

Reply:
Personal preference, as well as consistency with how we would refer to these in natural language (i,e, Serpent depletion code becomes SerpentDepcode rather than DepcodeSerpent)